pom.xml

```diff
@@ -276,34 +274,45 @@
         <developer>
             <id>matthieu</id>
             <name>Matthieu Baechler</name>
-            <email>matthieu@apache.org</email>
```

Comment:
please don't change that on my behalf. I'm not part of Linagora and I do contribute with Baechler Craftsmanship organization.